aws/services/lb/resource.ftl

```diff
@@ -203,6 +203,18 @@
         [#local protocol = port.Protocol]
     [/#if]
 
+    [#local certificateArn = getReference(certificateId, ARN_ATTRIBUTE_TYPE, regionId)]
```

Comment:
Should probably make this getExistingReference instead of getReference, otherwise it could return the ref object I think

Reply:
Cheers, sorted in 32b59f9